topics/proteomics/tutorials/metaquantome-function/workflows/main_workflow.ga

```diff
@@ -0,0 +1,623 @@
+        "0": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_db/metaquantome_db/2.0.0-0",
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [],
+            "label": null,
+            "name": "metaQuantome: database",
+            "outputs": [
+                {
+                    "name": "outfile",
+                    "type": "tar"
+                }
+            ],
+            "position": {
+                "left": 240.08682250976562,
+                "top": 168.95834922790527
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_db/metaquantome_db/2.0.0-0",
+            "tool_shed_repository": {
+                "changeset_revision": "7c45aa1c9464",
+                "name": "metaquantome_db",
+                "owner": "galaxyp",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"__input_ext\": \"data\", \"chromInfo\": \"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\", \"databases\": [\"ncbi\", \"go\", \"ec\"], \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": null,
+            "type": "tool",
+            "uuid": "cf54f9e8-832a-49cc-bdd4-543b07d69e32",
+            "workflow_outputs": [
+                {
+                    "label": null,
+                    "output_name": "outfile",
+                    "uuid": "d5c48127-549b-4f27-90e8-a4513012a62e"
+                }
+            ]
+        },
+        "1": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_sample/metaquantome_sample/2.0.0-0",
+            "errors": null,
+            "id": 1,
+            "input_connections": {},
+            "inputs": [],
+            "label": null,
+            "name": "metaQuantome: create samples file",
+            "outputs": [
+                {
+                    "name": "samples_file",
+                    "type": "tabular"
+                }
+            ],
+            "position": {
+                "left": 246.0937728881836,
+                "top": 315.0000057220459
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_sample/metaquantome_sample/2.0.0-0",
+            "tool_shed_repository": {
+                "changeset_revision": "49a957fb353a",
+                "name": "metaquantome_sample",
+                "owner": "galaxyp",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"samps_args\": {\"samps_src\": \"build\", \"__current_case__\": 0, \"samps\": [{\"__index__\": 0, \"group_name\": \"T2\", \"col_names\": \"T2_A1,T2_B1\"}, {\"__index__\": 1, \"group_name\": \"T4\", \"col_names\": \"T4A_1,T4B_1\"}]}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
The workflow is missing declaring the T7 samples. This means that all downstream steps that use T7 samples or columns as input fail. Please add the T7 sample in the sample creation step.